src/components/Card/Card.js

```diff
@@ -1,20 +1,22 @@
 import React, { useState } from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
-import Badge from '@material-ui/core/Badge'
-import CloseIcon from '@material-ui/icons/Close'
-import { makeStyles } from '@material-ui/core/styles'
-import MoreVertIcon from '@material-ui/icons/MoreVert'
-import AnnouncementIcon from '@material-ui/icons/Announcement'
-import DragIndicatorIcon from '@material-ui/icons/DragIndicator'
-import ChevronLeftIcon from '@material-ui/icons/ChevronLeft'
-import ChevronRightIcon from '@material-ui/icons/ChevronRight'
-import SaveIcon from '@material-ui/icons/Save'
-import IconButton from '@material-ui/core/IconButton'
-import SaveOutlinedIcon from '@material-ui/icons/SaveOutlined'
-import VisibilityIcon from '@material-ui/icons/Visibility'
-import VisibilityOffIcon from '@material-ui/icons/VisibilityOff'
-import MinimizeIcon from '@material-ui/icons/Minimize'
+import {
+  Badge,
+  IconButton
+} from '@mui/material'
+import CloseIcon from '@mui/icons-material/Close'
+import { makeStyles } from '@mui/styles'
+import MoreVertIcon from '@mui/icons-material/MoreVert'
```

Comment:
These can be combined just like the `@mui/material`

```js
import {
  MoreVert,
  Announcement,
  DragIndicator,
  ...
} from '@mui/icons-material';

Reply:
Combined them. At first I didn't do it because I thought I would need to update all the code below to reference the new name but then I just used MoreVert as MoreVertIcon